docs/documentation/guidance/rest-apis.md

```diff
@@ -8,6 +8,16 @@ When creating [REST APIs](https://en.wikipedia.org/wiki/Representational_state_t
 
 There are implementations available for many languages like C#, including low-level tooling, editors, user interfaces, code generators, etc. Here you can find a list of known tooling for the different languages: [OpenAPI-Specification/IMPLEMENTATIONS.md](https://github.com/OAI/OpenAPI-Specification/blob/main/IMPLEMENTATIONS.md).
 
-More information:
+## Using Microsoft TypeSpec
+
+While the OpenAPI Specification (OAS) is a popular method for defining and documenting RESTful APIs, there are other languages available that can simplify and expedite the documentation process. Microsoft TypeSpec is one such language that allows for the description of cloud service APIs and the generation of API description languages, client and service code, documentation, and other assets.
```

Comment:
Perhaps add a link here to Open API?

Reply:
At first I did not add them because OAS was mentioned before in the previous section, but I think there is no harm on it. Added links.